src/ui/source_editor_find_area.ui

```diff
@@ -19,7 +19,7 @@
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
-  <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1,1,0">
+  <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,0,1,0,0,0">
```

Comment:
I think the *old* value should have been `6,1,1,1`, and since you have added another `QLineEdit` and a `QPushButton`  I think the stretches should be revised to something like (though you might have to take the 5's down even more)...
```suggestion
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,1,5,1,1">
```
thinking about it, given that the buttons do not need to stretch (IIRC) you might even try: `"5,0,0,5,0,0"`!

Reply:
It effectively makes both text boxes the same size so I'm leaving it at "5,0,0,5,0,0", thanks!